index.html

```diff
@@ -1412,6 +1412,153 @@ <h3>Biometrics</h3>
+        activities. As with all things, strong consideration for the appropriate and ethical 
+        use of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+        <p>
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies.  
+          Avoidance of excess energy use that does not further other equally important 
+          goals such as human rights or personal privacy, particularly when that energy 
+          is sourced from non-sustainable energy creation methods, is a best 
+          practice that serves the interest of all.
+        </p>
+  
+        <p>
+          Direct comparison of each factor, and the balances between them, should be
+          considered when evaluating particular methods or approaches. Notably, the
+          ability to misuse or control DIDs that could impede privacy might be
+          found to be inversely correlated with additional energy consumption.
+        </p>
+        
+        <p class="advisement">
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />As part of following this principle,    
+          utilizing or authoring DID methods that require unsustainably-sourced 
+          energy as part of their technical implementation or utilization should
+          be very carefully considered in balance against alternative approaches.
+        </p>
+  
+        <p>
+          When selecting a DID method, the method that minimizes energy consumption 
```

Comment:
This entire paragraph should be removed.

Minimizing energy consumption is not the point of identifier technology. Rather, it is a political opinion with which some people have incorrectly maligned PoW networks. One should choose the energy consumption profile that adequately addresses ones needs, and to do so as sustainably as possible. 
This entire argument has also missed the very real fact that not only is bitcoin mining attracted to the lowest cost energy sources (which are increasingly sustainable), mining capacity is often added to energy systems that have unused capacity, enabling economic capture of of the excess energy produced when water that falls over the damn or wind that turns a turbine when demand is low. The net result is an increased profitability from these types of low-variance energy sources, which further decreases the cost of sustainable electricity, which further grows the percentage of energy WORLDWIDE that is sustainably produced. Shifting the break even point of new sustainable energy systems like wind farms, geo, hydro, and solar is already producing a shift in energy development and investment toward sustainable systems. Contrary to the disinformation narrative, there is a very real and non-zero increase in sustainable energy capacity precisely because PoW can turn fallow electrical production into positive economic returns.

In short, PoW networks already exist, they can be, and are, sustainably (as well as unsustainably) mined, and have a non-zero but hard to quantify positive impact on global, sustainable energy production, while providing a global service that many agree is worth the expense. Given that these networks already exist, utilizing them for globally resolvable proof of control for cryptographic identifiers is not a significant marginal impact on our environment.

Furthermore, DID methods, such as Ion make it possible to execute as many as 35,000 update transactions per second without requiring any additional energy expenditure (that is the total addressable opportunity for did:ion under the current specification). 

I challenge anyone who claims that PoW is a waste of energy to demonstrate how you get a globally distributed transaction layer at that volume with less energy and without a central authority.

As such, arguing that "energy minimization" should "generally be chosen" is exactly the kind of politically manipulative language that needs to be excised from this implementation guide. 

If all we wanted was to minimize energy, then we could just centralize everything into a single, bespoke asynchronous ASIC-based system that we all simply trust to be accurate.

But that's not what we are doing.

Centralizing *can* give you incredible economies of scale. That's why we have centralized systems.

But that's not what we are building here. As such, the innate hatchet job of this PR remains, IMO, completely inappropriate.